codebuild_specs/pr_workflow.yml

```diff
@@ -14,6 +14,10 @@ env:
 batch:
   fast-fail: false
   build-graph:
+    - identifier: build_tests_standalone
+      buildspec: codebuild_specs/build_tests_standalone.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
```

Comment:
if all the builds use the same `compute-type` env, can we somehow reduce the duplication?

Reply:
That's a good point, but I'd try this in separate PR since it's going to be quite a large change.
The goal for this PR is to add missing jobs.